common/presenters/table/object-to-table-head.test.js

```diff
@@ -18,7 +25,10 @@ describe('table head presenter', function() {
     const output = schemaExample.map(objectToTableHead)
     expect(output).to.deep.equal([
       {
-        html: 'move_type::label',
+        text: 'move_type::label',
```

Comment:
It doesn't really feel like the update to this test file explains well the new behaviour. 

The intended behaviour is a little hidden. It would be good to have separate tests using `context` to describe the two scenarios:

```javascript
context('with text content', function() {
  ...
})

context('with html content', function() {
  ...
})

context('with attributes', function() {
  ...
})
```